### ISSUE TITLE
\label{} causes the equation to break Tex

### ISSUE THREAD

**Issue by rylan** [mathjax/MathJax#1342]
### Issue Summary
When testing out the \label{some_lable} inside my math equations the equations breaks. If I paste the equation it works, but if I make any edits (including whitespace) the equation shows as broken

### Steps to Reproduce
Using the live demo from the https://www.mathjax.org/ site, paste the following equation 
```
\begin{equation}
  \int_0^\infty \frac{x^3}{e^x-1}\,dx = \frac{\pi^4}{15}
  \label{eq:sample}
\end{equation} 
```

Should see 
![equation](https://cloud.githubusercontent.com/assets/209707/11938077/13e7a904-a7e6-11e5-832f-7b0f5143c9d8.png)

Next remove some whitespace, the equation will not rerender ( will see 
![equation2](https://cloud.githubusercontent.com/assets/209707/11938106/56d1fb34-a7e6-11e5-9903-1ef0ec919d2e.png)
 ) 

remove \label and the function rerenders correctly

example is from 
https://docs.mathjax.org/en/v2.6-latest/tex.html?highlight=label.

**Comment by dpvc** [comment]
This is an issue with the demo page rather than MathJax itself.  Since labels must be unique on a page, MathJax complains when they are used a second time.  Since the demo page typesets the equation multiple times in the same page, the labels technically are repeated, and so the second time it causes an error.  The demo page probably should reset the labels between typesets.

**Comment by rylan** [comment]
Once again thank you @dpvc that is a huge help.
